unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/EmoteClipData.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+
+namespace DCL.Emotes
+{
+    public class EmoteClipData
+    {
+        public readonly AnimationClip Clip;
```

Comment:
small change, even if it's a pubic readonly variable it should start with lowercase c for our standard